src/gcp/cloudscheduler.ts

```diff
@@ -12,50 +13,109 @@ export interface Schedule {
     uri: string;
     httpMethod: string;
   };
+  retryConfig?: {
+    retryCount?: number;
+    maxRetryDuration?: string;
+    minBackoffDuration?: string;
+    maxBackoffDuration?: string;
+    maxDoublings?: number;
+  };
 }
-
-export function createJob(schedule: Schedule): Promise<void> {
+/*
+* createJob creates a cloudScheduler job
+* @returns a response from the cloud scheduler api
```

Comment:
Not sure if we need @returns (the other jsdocs don't have it and it's just in the description). I would remove it for consistency. Also we need @param